cyclonedx/model/__init__.py

```diff
@@ -28,6 +33,8 @@
 from a `cyclonedx.parser.BaseParser` implementation.
 """
 
+LOCALE_TYPE_REGEX = re.compile(r'^([a-z]{2})(-[A-Z]{2})?$')
```

Comment:
is this `LOCALE_TYPE_REGEX` really a public property?

Reply:
Updated.